packages/blockchain-wallet-v4-frontend/src/data/components/nfts/sagas.ts

```diff
@@ -354,6 +356,7 @@ export default ({ api, coreSagas, networks }: { api: APIType; coreSagas; network
       const retailToken = yield call(generateRetailToken)
       const order = yield call(fulfillNftOrder, { buy, gasData, signer })
       yield call(api.postNftOrder, order, action.payload.asset.collection.slug, guid, retailToken)
+      yield put(A.setSellOrder({}))
```

Comment:
We probably don't need this, as closeNftOrderFlow handles reseting of orderFlow state

Reply:
It wasn't resetting so thats why I added

Reply:
```
nftOrderFlowClose: (state) => {
      state.orderFlow = initialState.orderFlow
    },
```

maybe a bigger issue, is anything resetting?